tests/e2e/helpers/dumpinfo.go

```diff
@@ -0,0 +1,199 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package helpers
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/cilium/tetragon/tests/e2e/checker"
+	"github.com/cilium/tetragon/tests/e2e/flags"
+	"github.com/cilium/tetragon/tests/e2e/state"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/e2e-framework/klient/k8s/resources"
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+)
+
+var (
+	TetragonContainerName = "tetragon"
+	TetragonJsonPathname  = "/var/run/cilium/tetragon/tetragon.log"
```

Comment:
Hmm maybe /var/run/tetragon? is there a reason for having cilium there ?

Reply:
This is specific to our Helm chart. If we want to change it here we have to change it there first.